gradle-conjure/src/main/java/com/palantir/gradle/conjure/ExtractExecutableTask.java

```diff
@@ -48,12 +50,15 @@
 public abstract class ExtractExecutableTask extends Sync {
```

Comment:
FLUP, this should just be an artifact transform. I think we probably already have one, see `UnpackingArtifactTransformAction` from internal repository perhaps. You'd have to verify the output is the same or just adapt the output correctly